kevm_pyk/setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = kevm_pyk
+version = 0.0.1
+
+[options]
+python_requires = >= 3.8.*
+packages = find:
+package_dir =
+    =src
+install_requires =
+    pyk @ git+https://github.com/runtimeverification/k.git@v5.3.4#subdirectory=pyk
```

Comment:
```suggestion
    pyk @ git+https://github.com/runtimeverification/k.git@${somehow get this from deps/k}#subdirectory=pyk
```

Reply:
This is not trivial. Created an issue to log this for later: https://github.com/runtimeverification/erc20-verification/issues/389